docs/reference/modules/scripting/security.asciidoc

```diff
@@ -109,7 +110,8 @@ script.allowed_types: inline <1>
 
 By default all script contexts are allowed to be executed.  This can be modified using the
 setting `script.allowed_contexts`.  Only the contexts specified as part of the setting will
-be allowed to be executed.
+be allowed to be executed.  To specify no contexts are allowed, `set scripts.allowed_contexts`
```

Comment:
The backtick is in the wrong place, should be around `scripts.allowed_contexts`, not `set scripts.allowed_contexts`.